### PR TITLE
WIP: feat: add add api for post employee (error on run)

### PR DESCRIPTION
**Pull request by talarKoudoussian**

### CODE REVIEW

File: src/main/java/employee/data/Employee.java [hunk 1]
```diff
@@ -11,16 +9,20 @@
...
     private String lastName;
+
     @Column(name="added_date")
     private String addedDate;
+
     @Column(name="employment_status")
     private boolean employmentStatus;
 
-    public long getEmployeeId() {
+    public Long getEmployeeId() {
```
Comment (george-haddad) [inline src/main/java/employee/data/Employee.java]: revert this back to primitive, original variable is a primitive and this is boxing it back into an Object

**Comment by george-haddad** [comment]
You have now some duplicate code. `models/Employee.java` and `employee/data/Employee.java` are the same. You need to remove the `models` folder.  Also would be a good idea to remove `hello` folder too

File: db/employees.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "employee_id": "3",
+  "first_name": "Albert",
+  "last_name": "Einstein",
+  "added_date": "2018-05-20T00:00:00.000",
+  "employment_status": "1"
+}
```
Comment (george-haddad) [inline db/employees.json]: Lets update this `json` model with camelCase names. Camel case names are json convention too.

Comment (talarKoudoussian) [reply db/employees.json]: Should I leave the employeeId or is it better to remove it?

Comment (george-haddad) [reply db/employees.json]: keep it because it is a model of what can be there. You can put a comment saying that this value is omitted on POSTs

**Review by george-haddad — APPROVED** [review]
👍

**Pull request merged**